src/main/java/pl/javastart/task/ArrayUtils.java

```diff
@@ -0,0 +1,53 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class ArrayUtils {
+
+    private Scanner scanner = new Scanner(System.in);
+    private int[][] array;
```

Comment:
Wprowadzając pole (a przez to "stanowość") narażasz się na sytuację, w której ktoś wywoła metody nie w tej kolejności, której się spodziwałeś. Co jeśli `printArray` zostanie wywołane przed `createArray()`? Rozwiązaniem może być pozbycie się tego pola i zmiana sygnatury `createArray()`, tak, żeby przyjmowała tę tablicę.